client/coral-framework/graphql/mutations/index.js

```diff
@@ -10,16 +10,59 @@ export const postComment = graphql(POST_COMMENT, {
   options: () => ({
     fragments: commentView
   }),
-  props: ({mutate}) => ({
-    postItem: ({asset_id, body, parent_id} /* , type */ ) => {
-      return mutate({
+  props: ({ownProps, mutate}) => ({
+    postItem: ({asset_id, body, parent_id}) =>
+      mutate({
         variables: {
           asset_id,
           body,
           parent_id
+        },
+        optimisticResponse: {
+          createComment: {
+            comment: {
+              user: {
+                id: ownProps.auth.user.id,
+                name: ownProps.auth.user.username
+              },
+              created_at: new Date(),
+              body,
+              parent_id,
+              asset_id,
+              action_summaries: [],
+              tags: [],
+              status: null,
+              id: `${Date.now()}_temp_id`
```

Comment:
since this whole object is initialized here, does that mean that every time the Comment schema changes we're going to have to remember to update this?